libs/sdk-ui-kit/package.json

```diff
@@ -113,6 +115,7 @@
         "react": "^16.10.0",
         "react-dom": "^16.10.0",
         "ts-jest": "^26.3.0",
-        "typescript": "4.0.2"
+        "typescript": "4.0.2",
+        "@types/react-portal": "~4.0.2"
```

Comment:
Please use ^ for the dependencies. There is a `--caret` switch for the `rush add` command.

Reply:
Changed.